resource_customizations/argoproj.io/Rollout/health_test.yaml

```diff
@@ -1,4 +1,8 @@
 tests:
+- healthStatus:
```

Comment:
Am I an idiot - is this the test?

Reply:
Yes this is the test. We have that documented here: https://argoproj.github.io/argo-cd/operator-manual/health/#way-2-contribute-a-custom-health-check but I can see how that can be confusing. Honestly, I think we may want to rethink this testing framework. The current tests are valuable but I found it difficult to maintain.

Reply:
cool - you still need to sync with master